rules/builtins.build_defs

```diff
@@ -263,16 +263,21 @@ def licenses(licences):
     assert CONFIG.BAZEL_COMPATIBILITY, 'licenses() can only be called in Bazel compat mode'
     package(default_licences = licences)
 
-
 def json(value) -> str:
     """Returns a JSON-formatted representation of a plz value"""
     pass
 
-
 def breakpoint():
     """Breaks into an interactive debugging session."""
     pass
 
+def is_debug_label(name:str) -> bool:
```

Comment:
This provides a more ergonomic mechanism for rule authors to modify their targets when they match the target passed to `plz debug`, if they want to support debuggable targets.

This also fixes a previous issue where the only check being done was through `CONFIG.get("DEBUG")`, which meant unwanted targets were being modified as well, which was causing a few issues.